Dockerfile

```
@@ -0,0 +1,34 @@
+FROM ubuntu:latest
+
+RUN sysctl -w net.mptcp.enabled=1
+
+# Update package lists and install necessary dependencies
+RUN apt update && apt install -y \
```

```suggestion
RUN apt-get update && apt-get install -y \
```

`apt` is not designed to be used in scripts and automations.